manager/context.go

```diff
@@ -0,0 +1,80 @@
+package manager
+
+import (
+	"encoding/json"
+	"errors"
+	"net/http"
+
+	"gopkg.in/vinxi/vinxi.v0/plugin"
+	"gopkg.in/vinxi/vinxi.v0/rule"
+)
+
+// Context is used to share request context entities
+// across controllers.
+type Context struct {
+	Manager      *Manager
+	Scope        *Scope
+	Instance     *Instance
+	AdminPlugins *plugin.Layer
+	Request      *http.Request
+	Response     http.ResponseWriter
+	Rule         rule.Rule
+	Plugin       plugin.Plugin
+}
+
+// ParseBody parses the body.
+func (c *Context) ParseBody(bind interface{}) error {
+	if c.Request.Header.Get("Content-Type") != "application/json" {
+		c.SendError(415, "Invalid content type. Must be application/json")
+		return errors.New("Invalid type")
+	}
+
+	decoder := json.NewDecoder(c.Request.Body)
+	return decoder.Decode(&bind)
+}
+
+// Send is used to serialize and write the response as JSON.
+func (c *Context) Send(data interface{}) {
+	c.SendStatus(200, data)
+}
+
+// SendStatus is used to serialize and write the response as JSON with custom status code.
+func (c *Context) SendStatus(status int, data interface{}) {
```

Comment:
I think it would be more clear to have `SendOk(data interface{})` and `Send(status int, data interface{})` as `SendStatus` does not have a clear intent.